projects/storefrontlib/src/lib/store-finder/services/google-map-renderer.service.ts

```diff
@@ -134,10 +124,16 @@ export class GoogleMapRendererService {
   }
 
   /**
-   * Moves all the markers to the given map
-   * @param map {@link google.maps.Map} the map where all the markers will be moved. Pass null if you just want to erase markers
+   * Initialize and draw the map
+   * @param locations array of locations to be displayed on the map
```

Comment:
you're missing a param for the HTMLElement, we also usually don't put comments in this project